storage-transcribe-audio/functions/src/index.ts

```diff
@@ -1,9 +1,141 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
 import * as functions from "firebase-functions";
+import * as admin from "firebase-admin";
+import { getEventarc, Channel } from "firebase-admin/eventarc";
+import * as speech from "@google-cloud/speech";
+import * as path from "path";
+import * as os from "os";
+
+import * as logs from "./logs";
+import {
+  publishFailureEvent,
+  errorFromAny,
+  publishCompleteEvent,
+} from "./util";
+import mkdirp = require("mkdirp");
+import {
+  transcodeToLinear16AndUpload,
+  transcribeAndUpload,
+} from "./transcribe-audio";
+
+admin.initializeApp();
+
+const eventChannel: Channel | null = process.env.EXT_SELECTED_EVENTS
+  ? getEventarc().channel(process.env.EXT_SELECTED_EVENTS, {
+      allowedEventTypes: process.env.EXT_SELECTED_EVENTS,
+    })
```

Comment:
This doesn't look right, can you verify the arguments for this method? I think it just needs the second argument.

Reply:
Ah, you're right, it was wrong. Does the updated version look better? I'm basing it off of image-resize.

Reply:
I don't really know. Is there someone we could ask about this?

Reply:
Also, this doesn't seem to be reflecting the latest version of the file; that's very confusing...

Reply:
This is the practice followed by the Resize Images extension

Reply:
Resolving, we can have further discussion in a subsequent PR.